src/AzureIoTHub.Portal/Client/Pages/Index.razor

```diff
@@ -9,6 +8,6 @@
     protected override void OnInitialized()
     {
         // Go back to the list of device
-        NavManager.NavigateTo("devices");
+        NavManager.NavigateTo("dashboard");
```

Comment:
Why not having the dashboard directly on index page?

Reply:
I embedded the dashboard component within the index page